suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -725,15 +725,26 @@ export const getMainnets = (debug = false, bnb = false) =>
             (bnb || n.symbol !== 'bnb'),
     );
 
-export const getTestnets = (debug = false) =>
+export const getTestnetsCompatible = (debug = false) =>
     networksCompatibility.filter(
         n => !n.accountType && n.testnet === true && (!n.isDebugOnlyNetwork || debug),
     );
 
-export const getAllNetworkSymbols = () => networksCompatibility.map(n => n.symbol);
+const networksCollection: Network[] = Object.values(networks);
+
+export const getMainnets = (debug = false, bnb = false) =>
+    networksCollection.filter(
+        n => !n.testnet && (!n.isDebugOnlyNetwork || debug) && (bnb || n.symbol !== 'bnb'),
+    );
+
+export const getTestnets = (debug = false) =>
+    networksCollection.filter(n => n.testnet === true && (!n.isDebugOnlyNetwork || debug));
+
+export const networkSymbols = networksCollection.map(n => n.symbol);
 
-export const getEthereumTypeNetworkSymbols = () =>
-    networksCompatibility.filter(n => n.networkType === 'ethereum').map(n => n.symbol);
+export const ethereumTypeNetworkSymbols = networksCollection
+    .filter(n => n.networkType === 'ethereum')
+    .map(n => n.symbol);
```

Comment:
- It does not have to be a function, as its ouptut was strictly constant
- Also it did not work properly, it returned array of non-unique symbols, now they're unique
  - The function was used, but making it unique won't break anything: 
  - All usages are only to query for `ethereumTypeNetworkSymbols.includes`
  - that doesn't matter if it's unique or non-unique  :heavy_check_mark: